jbrowse/config/dev.config.js

```diff
@@ -25,7 +25,21 @@ clientConfig.resolve.fallback =
 
 clientConfig.plugins =
     [new webpack.ProvidePlugin({regeneratorRuntime: 'regenerator-runtime'}),
-     new NodePolyfillPlugin()].concat(constants.processPlugins(entryPoints))
+     new NodePolyfillPlugin(),].concat(constants.processPlugins(entryPoints))
```

Comment:
is there some typescript convention around trailing commas i dont know about? should this be here?

Reply:
No, it's left over. Good catch. Removed